osmaxx-py/excerptconverter/gisexcerptconverter/gis_excerpt_converter.py

```diff
@@ -223,19 +232,23 @@ def execute_task(extraction_order_id, supported_export_formats, execution_config
                         )
 
                 elif type(bounding_geometry) == models.OsmosisPolygonFilterBoundingGeometry:
+                    message = _('GIS excerpt converter is not yet able to extract polygon excerpts.')
                     converter_helper.inform_user(
                         messages.ERROR,
-                        _('GIS excerpt converter is not yet able to extract polygon excerpts.'),
+                        message,
                         email=False
                     )
-
+                    logging.error(message)
                 else:
+                    message = _('GIS excerpt converter is not yet able to extract excerpts of type {type}.').format(
+                        type=type(bounding_geometry).__name__
+                    )
                     converter_helper.inform_user(
                         messages.ERROR,
-                        _('GIS excerpt converter is not yet able to extract excerpts of type %s.') %
-                        type(bounding_geometry).__name__,
+                        message,
                         email=False
                     )
+                    logging.error(message)
```

Comment:
And please log what crazy type of bounding_geometry we got to fall in else (This should never happen :wink: ).